build_projects/shared-build-targets-utils/VersionRepoUpdater.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.DotNet.Cli.Build
     public class VersionRepoUpdater
     {
         private static Regex s_nugetFileRegex = new Regex("^(.*?)\\.(([0-9]+\\.)?[0-9]+\\.[0-9]+(-([A-z0-9-]+))?)\\.nupkg$");
-
+        private static string s_baseGitHubPath = $"/build-info/dotnet/cli";
```

Comment:
We need to keep "CLI" specific concepts out of this class, as it is duplicated between both the 'cli' and 'core-setup' repo, and @brthor wants to keep the files exactly the same.

Also, I imagine other teams copying this file into their own repos in order to update 'dotnet/versions' with their build info.  It'd be easier to dupe if it is as generic as possible.